probe-rs-tools/src/bin/probe-rs/cmd/info.rs

```diff
@@ -553,8 +553,13 @@ fn process_component_entry(
             let ApAddress::V2(addr) = access_port.ap() else {
                 unreachable!("This should only happen on ap v2 addresses.");
             };
-            let addr = addr.clone().append(component.id().component_address());
-            let addr = FullyQualifiedApAddress::v2_with_dp(dp, addr);
+            if addr.0.is_some() {
+                unimplemented!("Nested memory APs are not yet supported.");
```

Comment:
I would rather this returned a proper error rather than panicking :)

Reply:
Fixed this and a few other cases for the STLink and BMP probes APv2 support to return proper error codes in 6714e531